### DISCUSSION TITLE
Issues with RBAC previews

### DISCUSSION THREAD

**Discussion by AlexandriaLittle** [Budibase/budibase#6177] [Help]
Hi All,

I'm trying to build an app with a few different roles. For some reason, however, my / screen of a role derived from basic is appearing as my admin / screen. Nothing I'm trying seems to work to get it to display the correct screens for the Admin Role. I attached the app so you guys can look at it.

[Requisition App Draft Three-export-1654202635750.txt](https://github.com/Budibase/budibase/files/8827698/Requisition.App.Draft.Three-export-1654202635750.txt)

I'm seeing the Purchasing Coordinator role's home screen as the Admin home screen in preview.

**Comment by shogunpurple** [comment]
Hi @AlexandriaLittle - I cannot seem to reproduce this one, it's working as expected with your app export. 

- Which version of budibase are you on?
- Cloud/Self host? 

I'll see if I can recreate inside your exact environment.

**Reply by AlexandriaLittle** [reply]
I'm self-hosting on version 1.0.194

**Reply by AlexandriaLittle** [reply]
I'm still grappling with the same problem. :/ Is there something I can do in my environment to help? It seems like the behavior where a higher role overrides the screens of lower roles is broken.
